update to ember-cli 4.11

tzellman opened the pull request:
- updated app to latest ember-cli/ember source and addons
- fixed linting errors
- fixed test errors

Commit:
chore: ember-cli-update to latest 4.11 release

- also, updated all dependencies/addons to latest

Commit:
chore: fixed linting errors

- fixed prettier linting errors
- removed {{yield}}-only templates - not allowed
- also added constructor to modifier

Commit:
fix: fixed tests

- canvas items need the wrapping canvas to render
- default the text args to an empty string to avoid a fabric Error
- skip the fabric-register-canvas test for now

hilljh82 commented:
@tzellman Quick comment. Can you leave the space between the identifier and open parenthesis that you removed to fix linting issues? That is the coding style of our organization.

hilljh82 commented:
Also, we don't use prettier in our repositories because we don't agree with their styling

Commit:
chore: remove prettier and conform to style requirements

tzellman commented:
> @tzellman Quick comment. Can you leave the space between the identifier and open parenthesis that you removed to fix linting issues? That is the coding style of our organization.

No problem! I removed prettier from the linting process and added the spaces back.
